### PR TITLE
Responsive navbar.

### PR DESCRIPTION
**Pull request by haaarshsingh**
Current state:
image

The navbar's donate button was removed. I will create a separate pull request with a more fancy donate button on the header. 

Previous state:
image

### CODE REVIEW

File: public/style/components/navigation.css [hunk 1]
```diff
@@ -57,11 +56,11 @@ nav ul li.donate a:hover
...
-        top: 5px;
+@media only screen and (max-width: 600px) {
+    nav {
+        font-size: 17px;
     }
-}
+    nav ul {
+        padding: 0.3rem 0.4rem;
+    }
+}
```
Comment (CodeMyst) [inline public/style/components/navigation.css:66]: should follow the same style (brace on new line)

File: public/style/pages/home.css [hunk 2]
```diff
@@ -98,3 +98,10 @@
     font-size: var(--font-size-normal);
     width: 30%;
 }
+
+@media only screen and (max-width: 600px) {
+    #create-paste {
+        padding: 0.2rem 0.5rem;
+        font-size: 14px;
+    }
+}
```
Comment (CodeMyst) [inline public/style/pages/home.css:107]: again, style issues

**Review by CodeMyst — CHANGES_REQUESTED** [review]

**Comment by CodeMyst** [comment]
also not sure if this is the way to go, but for now this is better than it was.

File: public/style/components/navigation.css (hunk 1 above)
Comment (haaarshsingh) [reply public/style/components/navigation.css:66]: ok. Will be fixing :+1:

**Pull request closed without merging**